documentation/api/query/v4/event-counts.markdown

```diff
@@ -60,7 +61,13 @@ This endpoint builds on top of the [`events`][events] endpoint, and supports all
 ### Response format
 
 The response is a JSON array of maps. Each map contains the counts of events that matched the input
-parameters. The events are counted based on their statuses: `failures`, `successes`, `noops`, `skips`.
+parameters. The events are counted based on their statuses and corrective_change flags (PE only):
+`failures` (status = 'failure'),
+`successes` (status = 'success' and corrective_change = false),
+`noops` (status = 'noop' and corrective_change = false),
```

Comment:
@wkalt What kind of flag do you mean? A special URL parameter? That has been discussed earlier. I had the impression we wanted to avoid that.

Maybe, you mean something else here.